tests/PHPStan/Rules/PhpDoc/data/incompatible-throws.php

```diff
@@ -82,3 +82,10 @@ function notThrowableInUnionThrows()
 function notThrowableInIntersectThrows()
 {
 }
+
+/**
+ * @throws void
+ */
+function voidThrows()
```

Comment:
I'd also check what happens when `stdClass|void` and `\Exception|void` is used.

Reply:
Already added